### PR TITLE
Vates testing docs

### PR DESCRIPTION
**Pull request by keeeto**
**Description of work.**

This PR adds testing for the VATES interface. This testing will be done in each release cycle to ensure that the interface is in minimal required working condition. This means that all features that are regularly/routinely required by users should be tested and verified.

**Report to:** nobody. 

**To test:**

* Build the `dev-docs-html`
* Open `dev-docs/index.html` and navigate to `Testing` > `VATES Testing`
* Check that the tests make sense and cover all required features

*There is no associated issue.*

*This does not require release notes* because it is not a user-facing feature.
---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: dev-docs/source/Testing/VATES/VATESTEsting.rst [hunk 1]
```diff
@@ -0,0 +1,61 @@
+.. _vates_testing:
```
Comment (OwenArnold) [inline dev-docs/source/Testing/VATES/VATESTEsting.rst]: I will do a more thorough review when I'm back, but all references to "VATES" should be VSI. Vates was the original project name, which included visualisation and workspace extensions for multidimensional processing. VSI is vates-simple-interface, which we already refer to in all existing documentation. Personally I would prefer that this was called "n-d viewer" or something similar, but VSI is the best choice for now.

File: dev-docs/source/Testing/VSI/VSITesting.rst [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+- Make sure that you have the Mantid system test data. You can do this by building a SystemTestData target of Mantid.
+- Put the system test data directory in your Mantid user directories
+
+
+**Time required 20-30 minutes**
+
+--------------
+
+#. Load the script SXD_NaCl.py in MantidPlot. Run it.
+#. This should load a workspace called QLab. Right-click this workspace and choose 'Show VSI simple interface'.
```
Comment (OwenArnold) [inline dev-docs/source/Testing/VSI/VSITesting.rst]: Sorry to be pedantic, but the menu option is actually "Show VATES simple interface"

**Review by OwenArnold — CHANGES_REQUESTED** [review]
Apart from that minor correction looks very good. Would be even better as a squish test ;)

File: dev-docs/source/Testing/index.rst [hunk 3]
```diff
@@ -24,4 +24,5 @@ creation is outlined in :ref:`issue_tracking`.
    IndirectInelastic/IndirectInelasticAcceptanceTests
    ErrorReporter-ProjectRecovery/ErrorReporterTesting
    ErrorReporter-ProjectRecovery/ProjectRecoveryTesting
+   VATES/VATESTEsting
```
Comment (OwenArnold) [inline dev-docs/source/Testing/index.rst]: VSITesting?

**Comment by keeeto** [comment]
@OwenArnold - I think that all the changes you requested are made. Just requires a final ok from your end. Cheers.

**Review by OwenArnold — APPROVED** [review]

**Pull request merged**